lib/segment/src/index/field_index/bool_index/simple_bool_index.rs

```diff
@@ -170,10 +173,34 @@ mod memory {
             self.indexed_count
         }
 
+        pub fn iter_has_true_measured<'a>(
+            &'a self,
+            hw_counter: &'a HardwareCounterCell,
+        ) -> impl Iterator<Item = PointOffsetType> + 'a {
+            self.trues
+                .iter_ones()
+                .map(|v| v as PointOffsetType)
+                .measure_hw_with_cell_and_fraction(hw_counter, u8::BITS as usize, |i| {
+                    i.payload_index_io_read_counter()
+                })
+        }
+
         pub fn iter_has_true(&self) -> impl Iterator<Item = PointOffsetType> + '_ {
             self.trues.iter_ones().map(|v| v as PointOffsetType)
         }
 
+        pub fn iter_has_false_measured<'a>(
+            &'a self,
+            hw_counter: &'a HardwareCounterCell,
+        ) -> impl Iterator<Item = PointOffsetType> + 'a {
+            self.falses
+                .iter_ones()
+                .map(|v| v as PointOffsetType)
+                .measure_hw_with_cell_and_fraction(hw_counter, u8::BITS as usize, |i| {
```

Comment:
Fot this we should have 1/8 of a byte per value, not the other way around

Reply:
the `*_and_fraction` function calculates `1/fraction` and not `*fraction`, it's a different function than in other places